crates/core/src/test_utils/execution_helpers.rs

```diff
@@ -0,0 +1,32 @@
+use reth_primitives::{BlockId, H256, U256};
+
+use super::deploy_helpers::{create_raw_ethereum_tx, KakarotTestEnvironmentContext};
+use crate::client::api::KakarotEthApi;
+use crate::models::felt::Felt252Wrapper;
+
+pub async fn execute_tx(env: &KakarotTestEnvironmentContext, contract: &str, selector: &str, args: Vec<U256>) -> H256 {
+    let contract = env.evm_contract(contract);
+    let contract_eth_address = {
+        let address: Felt252Wrapper = contract.addresses.eth_address.into();
+        address.try_into().unwrap()
+    };
+    let client = env.client();
+    let kakarot = env.kakarot();
```

Comment:
I believe we can use [`resources_with_contract`](https://github.com/kkrt-labs/kakarot-rpc/blob/main/crates/core/src/test_utils/deploy_helpers.rs#L754C4-L754C4) here as well

```rust
let (client, kakarot, contract, contract_eth_address) = env.resources_with_contract();
```

Reply:
ah damn you are right !